devops/jobs/MinosLifecycle.groovy

```diff
@@ -87,6 +88,7 @@ class MinosLifecycle {
                         env('ENVIRONMENT', environment)
                         env('DEPLOYMENT', deployment)
                         env('AWS_REGION',extraVars.get('AWS_REGION','us-east-1'))
+                        env('SNITCH', inner_config.get('snitch', ''))
```

Comment:
I'm fine with making this an optional setting but then the bash script should be updated.